Tensile/Tests/extended/direct_to_vgpr/dtv_hgemm.yaml

```diff
@@ -219,7 +219,7 @@ BenchmarkProblems:
         - PrefetchGlobalRead: [1]#[1,2]
         - PrefetchLocalRead: [2,3,5,9]
         - ScheduleIterAlg: [3]
-        - StaggerU: [0,32]
+        #- StaggerU: [0,32]
```

Comment:
Was this option disabled due to compatibility or to save test time?

Reply:
This is just to save time.
I cannot simply double the test parameters which significantly increase test time.
I do not think StaggerU makes big change here.